manifests/init.pp

```diff
@@ -170,6 +179,13 @@
   include certs::foreman_proxy
   Class['certs::foreman_proxy'] ~> Service['foreman-proxy']
 
+  if $reverse_proxy_real or !$shared_with_foreman_vhost {
+    class { 'certs::apache':
```

Comment:
I think this doesn't result in a duplicate class declaration with katello (since it should be scoped to just content proxies), but that'd be good to check.